app/models/drupal_node.rb

```diff
@@ -399,9 +418,9 @@ def totalcount
   # URL-related methods:
 
   # is this used anymore? deprecate?
-  def slug
-    self.path.split('/').last
-  end
+  # def slug
+  #  self.path.split('/').last
+  # end
 
```

Comment:
In the present context path is the full url of a node like `/notes/used/07-06-2016/some-test-url` while slug is only the parameterized version of title  like `some-test-url`